metadata-ingestion/src/datahub/ingestion/glossary/datahub_classifier.py

```diff
@@ -71,7 +71,7 @@ class Config:
 # TODO: Generate Classification doc (classification.md) from python source.
 class DataHubClassifierConfig(ConfigModel):
     confidence_level_threshold: float = Field(
-        default=0.6,
+        default=0.68,
```

Comment:
Just want to make sure this is intended

Reply:
yes, this to to keep the precision(lower confidence level) same as before, as confidence interval widens with less sample size.